### PR TITLE
docs: update test3 mentions, add test5 mentions

### PR DESCRIPTION
**Pull request by leohhhn**
## Description

This PR updates the mentions of test3 after its deprecation, and adds text on test5.

<details><summary>Contributors' checklist...</summary>

- [ ] Added new tests, or not needed, or not feasible
- [ ] Provided an example (e.g. screenshot) to aid review or the PR is self-explanatory
- [ ] Updated the official documentation or not needed
- [ ] No breaking changes were made, or a `BREAKING CHANGE: xxx` message was included in the description
- [ ] Added references to related issues and PRs
- [ ] Provided any useful hints for running manual tests
</details>

### CODE REVIEW

**Comment by Gno2D2** [comment]
I'm a bot that assists the Gno Core team in maintaining this repository. My role is to ensure that contributors understand and follow our guidelines, helping to streamline the development process.

The following requirements must be fulfilled before a pull request can be merged.
Some requirement checks are automated and can be verified by the CI, while others need manual verification by a staff member.

These requirements are defined in this [configuration file](https://github.com/gnolang/gno/tree/master/contribs/github-bot/internal/config/config.go).

## Automated Checks

 🟢 Maintainers must be able to edit this pull request
 🟢 The pull request head branch must be up-to-date with its base
 🔴 Changes to 'docs' folder must be reviewed/authored by at least one devrel and one tech-staff


## Manual Checks

*No manual checks match this pull request.*

<details><summary><b>Debug</b></summary><blockquote>
<details><summary><b>Automated Checks</b></summary><blockquote>

<details><summary>Maintainers must be able to edit this pull request</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Maintainer can modify this pull request

```
</blockquote></details>

<details><summary>The pull request head branch must be up-to-date with its base</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 On every pull request

```
### Then
```
🟢 Requirement satisfied
└── 🟢 Head branch (leohhhn:test3-docs) is up to date with base (master): behind by 0 / ahead by 1

```
</blockquote></details>

<details><summary>Changes to 'docs' folder must be reviewed/authored by at least one devrel and one tech-staff</summary><blockquote>

### If
```
🟢 Condition met
└── 🟢 A changed file matches this pattern: ^docs/ (filename: docs/concepts/testnets.md)

```
### Then
```
🔴 Requirement not satisfied
└── 🔴 Or
    ├── 🔴 And
    │   ├── 🟢 Pull request author is a member of the team: devrels
    │   └── 🔴 At least 1 user(s) of the team tech-staff approved pull request
    └── 🔴 And
        ├── 🔴 Pull request author is a member of the team: tech-staff
        └── 🔴 At least 1 user(s) of the team devrels approved pull request

```
</blockquote></details>

</blockquote></details>



</blockquote></details>

File: docs/concepts/testnets.md
Comment (leohhhn) [inline docs/concepts/testnets.md:64]: This and the TImeliness of code item are not entirely correct, are they?

**Review by Kouteki — APPROVED** [review]
LGTM

File: docs/concepts/testnets.md
Comment (zivkovicmilos) [reply docs/concepts/testnets.md:64]: Drop the versioning for packages & Realms.

We will follow releases for test5, this is fine

**Review by zivkovicmilos — APPROVED** [review]

**Comment by notJoon** [comment]
Aproved by core members. so remove `review/triage-pending` flag

File: docs/concepts/testnets.md
Comment (leohhhn) [reply docs/concepts/testnets.md:64]: [607b341](https://github.com/gnolang/gno/pull/3259/commits/607b34181a8bc6eecc1f0efc9a936d0a14adf75b)

**Pull request merged**
